libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
1.In the 'fetchQuote' method, this.storePickerFrom will return undefined, hence the code will break.Make sure we have correct reference to the storePickerForm when susbscribing to the value changes.

2.Make sure to unsubscribe to all the subscriptions in the form.

3.Make sure these changes have test case coverage.